<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSimSelect</class>
 <widget class="QMainWindow" name="frmSimSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>物理场选择</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxDomainPF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>求解类型</string>
    </property>
    <widget class="QRadioButton" name="rdb1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>电磁--电路</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>热</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>结构</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>热--结构</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>热--结构--电磁--电路</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>结构--电磁--电路</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>电磁--电路--热</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdb8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>电磁--电路--热--结构</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
